cpp/src/cider/tests/utils/CiderNextgenTestBase.h

```diff
@@ -80,6 +80,11 @@ class CiderNextgenTestBase : public testing::Test {
   cider::exec::nextgen::context::CodegenOptions codegen_options_ = {};
 };
 
+class CiderNextgenTestBaseWithoutDuckDB : public CiderNextgenTestBase {
```

Comment:
rename to `CiderStandaloneNextgenTestBase` would be better?

Reply:
sure, I will modify it in the next PR.